python/langsmith/client.py

```diff
@@ -491,6 +504,20 @@ def __init__(
         # Create a session and register a finalizer to close it
         session_ = session if session else requests.Session()
         self.session = session_
+        self.compress_traces = ls_utils.get_env_var("USE_RUN_COMPRESSION")
+        if self.compress_traces:
+            self._futures: set[cf.Future] = set()
```

Comment:
ooc: why can't we re-use the tracing queue property to support both for this? I'm still reading, but we just increased the number of properties by ~30%

Reply:
I think it's also better to keep them separate since they are different types and I'd be confused if I started using compression and a property completely changed. Additionally, the different typing caused mypy errors within the client when I tried to keep them in the same property. I've removed a significant amount of other properties that were unnecessary.